### ISSUE TITLE
Which is the maximum dimension of the input file?

### ISSUE THREAD

**Issue by Antonio-Nappi** [wangk4/piRNN#2]
I'm trying to discover novel pirna with an input file of 600k reads, (1.2M lines) but the script aborts the computation, is it a problem of the input file, maybe because it's too large? My configuration is
   
- 8 GB of Ram
- 20 GB on the disk
- 8 core

    (It's an university project)

**Comment by wangk4** [comment]
Hi, 
did you tried piRNN with a small input file? Please make sure the input file is in fasta format and if a small input files works fine, that might be the size of the input caused this problem.

**Comment by Antonio-Nappi** [comment]
yes, I try it with the example and it works great, but with large file it aborts the computation

**Comment by wangk4** [comment]
i think you can just split your input file into some small files then just run it one by one. Maybe 100k read per file.
